### PR TITLE
Add content and tests for the ship class

### PR DESCRIPTION
**Pull request by carson-jardine**
Added methods and attributes for the Ship class and wrote out tests. Will try to refactor so tests are more organized. Any thoughts?

### CODE REVIEW

File: lib/ship.rb [hunk 1]
```diff
@@ -1 +1,19 @@
 #Ship Class
+require 'pry'
+class Ship
+  attr_reader :name, :length, :health
+
+  def initialize(name, length)
+    @name = name
+    @length = length
```
Comment (coreycartercodes) [inline lib/ship.rb:8]: I believe we'll want to remove the @ on the @health = @length variable due to scope if I understand it correctly.

File: test/ship_test.rb [hunk 2]
```diff
@@ -1 +1,58 @@
...
+    assert_equal 2, @cruiser.health
+  end
+
+  def test_sunk_after_hit
+    @cruiser.hit
+
+    assert_equal false, @cruiser.sunk?
+  end
+
+  def test_sunk_after_three_hit
```
Comment (coreycartercodes) [inline test/ship_test.rb:44]: Love the tests. You are way more thorough than I am at those.

**Review by coreycartercodes — CHANGES_REQUESTED** [review]
See comments. Thanks!

File: lib/ship.rb (hunk 1 above)
Comment (carson-jardine) [reply lib/ship.rb:8]: Just tested this and removing the @ and replacing it with just "length" seems to work! Scope is not my strong suit, thanks for catching that. Will push changes soon.

File: test/ship_test.rb (hunk 2 above)
Comment (carson-jardine) [reply test/ship_test.rb:44]: Thanks! I get anxious something is broken otherwise. I do want to break it down by category but that'll be a later project.

**Pull request merged**